libs/base/src/lib/context/NetworkProvider.tsx

```diff
@@ -148,8 +148,8 @@ const etherscanUrl =
     }
   }
 
-const graphMainnetEndpoint = (subgraphId: string, version: number, apiKey: string): string =>
-  `https://gateway.thegraph.com/api/${apiKey}/subgraphs/id/${subgraphId}-${version}`
+const graphMainnetEndpoint = (subgraphId: string, apiKey: string): string =>
+  `https://gateway.thegraph.com/api/${apiKey}/subgraphs/id/${subgraphId}`
```

Comment:
wouldn't it be better to directly call `process.env.NX_SUBGRAPH_API_KEY` and remove arg?

Reply:
we might use different api keys for different subgraphs, so i think keeping the arg here for flexibility makes sense